src/components/AppComponent.js

```diff
@@ -100,8 +100,8 @@ export const AppComponent = connect(({ dataNonce, focus, search, showContexts, u
     <HelperShortcuts />
     <HelperFeedback />
 
-    { // render as header on desktop
-    !isMobile ? <NavBar position='top' /> : null}
+    { // render as header on desktop and mobile
+    <NavBar position='top' /> }
```

Comment:
This will reposition the entire `NavBar`, however the issue is that just the status message needs to be repositioned.